.github/workflows/unit-test-on-pull-request.yml

```
@@ -56,7 +56,7 @@ jobs:
       fail-fast: true
       max-parallel: 2
       matrix:
-        go: ["1.22"]
+        go: ["stable"]
```

If resources (CI time/cost) allow it, it might be interessting to run the benchmarks on the minimal supported Go version as defined in `go.mod` as well on current stable. This way, improvements could also be shown.
```suggestion
        go: ["stable", "1.18.10"]
```

---

I get your point, but don't think there is any interest from our side in benchmarking Go 1.18. Users best do their own benchmarks with whatever Go versions they are going to use and with code focused on their requirements.